src/app.js

```diff
@@ -107,25 +110,59 @@ export class App {
         settings: 'fa fa-tachometer'
       },
       {
-        route: 'login', name: 'login', moduleId: PLATFORM.moduleName('./login'), nav: false, title: 'Login', settings: 'fa fa-sign-in'
+        route: 'login',
+        name: 'login',
+        moduleId: PLATFORM.moduleName('./login'),
+        nav: false,
+        title: 'Login',
+        settings: 'fa fa-sign-in'
       },
       {
-        route: 'register', name: 'register', moduleId: PLATFORM.moduleName('./register'), nav: false, title: 'Register', settings: 'fa fa-user-plus'
+        route: 'register',
+        name: 'register',
```

Comment:
@Unesdala please fix your editor so that these autoformat changes do not occur, also please put back this file as it was formated before, thanks